### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by hagan199**
Add work section
Add about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,11 +50,8 @@ <h1 class="intro">I’m Emmanuel Hagan Glad to see you!</h1>
 
       <!---End Social Link-->
     </ul>
-
-
   </section>
 
-
       <!-- Work section -->
     <section class="work">
```
Comment (danmainah) [inline index.html:56]: - [ ] Kindly add styling to your section container for your woks section, I can find it also ensure it has `display:grid` in it.

Comment (danmainah) [inline index.html:56]: - [ ] Kindly make sure your section has a title as indicated in the design.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @hagan199 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
- [ ] Kindly check on your about section design, its not aligned with the other elements as shown below:

image


- [ ] Kindly make sure your button design matches the design given when someone hovers or clicks the button in the section highlighted below:

image

This is the design required:
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**